code/signs-markings/signs-markings.js

```diff
@@ -187,20 +187,20 @@ $(document).on("knack-page-render.any", function(event, page) {
   }
 });
 
-$(document).on("knack-scene-render.scene_1014", function(event, page) {
+$(document).on("knack-scene-render.scene_1014", function() {
   // CSR issue - markings details
   // update iframe src from detail field
-  var iframe_url = $("span:contains('apps/webappviewer')").text();
```

Comment:
It looks like Knack made a subtle change to the way they structure their html, the result of which was that this was grabbing multiple urls and assigning the `iframe` href to something like this:

```
https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=c9bda776b1934e439285570b29d3259c&amp;level=18&amp;marker=3103784;10056745;102739;;;25-00219025https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=c9bda776b1934e439285570b29d3259c&amp;level=18&amp;marker=3103784;10056745;102739;;;25-00219025
```
This new selector is more specific and will hopefully be more resilient.